storage/remote/read_test.go

```diff
@@ -122,12 +123,12 @@ func TestSeriesSetFilter(t *testing.T) {
 			toRemove: model.LabelSet{"foo": "bar"},
 			in: &prompb.QueryResult{
 				Timeseries: []*prompb.TimeSeries{
-					{Labels: labelsToLabelsProto(labels.FromStrings("foo", "bar", "a", "b")), Samples: []prompb.Sample{}},
+					{Labels: labelsToLabelsProto(labels.FromStrings("foo", "bar", "a", "b")), Samples: samples},
```

Comment:
Instead of this, `Samples: nil` would work.

Reply:
haha, yes, it would work.

Reply:
Or even just omitting `Samples` completely, in which case it will take its zero value `nil` automatically.